routing/index_router.cpp

```diff
@@ -112,8 +130,8 @@ IRouter::ResultCode SingleMwmRouter::DoCalculateRoute(m2::PointD const & startPo
       IndexGraphLoader::Create(m_numMwmIds, m_vehicleModelFactory, m_estimator, m_index),
       m_estimator);
 
-  // TODO remove to activate CrossMwmGraph.
-  graph.BlockMwmBorders();
+  if (blockMwmBorders)
+    graph.SetBordersAreOpened(false);
```

Comment:
What about OpenBorders()/CloseBorders() instead of SetBordersAreOpened(true/false)?